Create initial spec for mapml2geojson.

prushforth opened the pull request:
- [x] mapml2geojson

Will add geojson2mapml in future PR

prushforth committed:
Create initial spec for mapml2geojson.

prushforth committed:
Revert inadvertent search-and-replace error

AliyanH commented on spec/index.html:
```diff
@@ -860,6 +867,69 @@ <h4>The <code>&lt;<dfn id="the-layer-element">layer</dfn>&gt;</code> element</h4
+          element does not conform to that structure, incorrect output MAY result.
+        </p>
+        <p>
+          The <a href="#dom-geojsongenerationoptions-transform"><code>transform</code></a> option, 
+          allows the author to trigger the reverse projection transformation of 
+          the coordinates of feature geometries when converting to the GeoJSON 
+          [[rfc7946]] format. By default 
+          (<a href="#dom-geojsongenerationoptions-transform"><code>transform</code></a> === <code>false</code>),
+          no reverse projection transformation of coordinates will be performed. 
```
With the current implementation, the transform option is set to 'true' by default, so it tries to do the reverse projection transformation of the coordinates, unless explicitly told not to through the transform=false option.

Should the implementation be changed (minor fix)?

prushforth replied:
I changed the description.  That should teach me to ad lib late in the afternoon.  Thanks for catching this!

prushforth committed:
Fix small things with layer element

AliyanH commented on spec/index.html:
```diff
@@ -786,11 +786,10 @@ <h4>The <code>&lt;<dfn id="the-layer-element">layer</dfn>&gt;</code> element</h4
-        <dd id="attr-layer-disabled"><code>disabled</code> — Sets the disabled state of the layer in the layer control.</dd>
-        <dd id="attr-layer-hidden"><code>hidden</code> — Visibility status of the layer in the layer control.</dd>
+        <dd id="attr-layer-src"><code>src</code> — The URL of the layer resource.</dd>
+        <dd id="attr-layer-label"><code>label</code> — A label for the layer in the layer control, if the layer is not not <code>hidden</code>. The map author's <code>title</code> element takes precedence over <code>label</code>, if available.</dd>
```
Add a link to the title element?

prushforth replied:
Done

prushforth committed:
Fix sectioning problem introduced I don't know how
Applied some validator.nu advice used to fix some markup errors
Removed comments to cut down document size put into 'cuts' file

prushforth committed:
Fix description of transform to conform to implementation (def===true)
